INSTALLATION.rst

```diff
@@ -448,7 +328,7 @@ directories. The reasons for this include:
 To avoid tampering with the system Python_ installation, you can employ one
 of the utilities that manage packages and their dependencies independently
 of the system package manager and the system directories.  These utilities
-include conda_, Stow_, Virtualenv_ and zc.buildout_, amongst others.
+include conda_, Nix_, Stow_, Virtualenv_ and zc.buildout_, amongst others.
```

Comment:
[Buildout](https://pypi.org/project/zc.buildout/) refers to itself as Buildout, not zc.buildout.

Reply:
Why?!?